### PR TITLE
Modules: Clarify Redis Stack support

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
Staged previews are available:

- [Redis Stack page](https://docs.redis.com/staging/jira-doc-1353/modules/redis-stack/)
- [Enterprise capabilities](https://docs.redis.com/staging/jira-doc-1353/modules/enterprise-capabilities/)
- [Redis Stack and modules](https://docs.redis.com/staging/jira-doc-1353/modules/) main page

File: content/modules/enterprise-capabilities.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ The following table shows which modules are supported by Redis Enterprise Softwa
...
 | [RedisJSON]({{<relref "/modules/redisjson">}})   | &#x2705; Yes | &#x2705; Yes |
-| [RedisGraph]({{<relref "/modules/redisgraph">}}) | &#x2705; Yes | &#x2705; Yes |
+| [RedisGraph]({{<relref "/modules/redisgraph">}})[^4] | &#x2705; Yes | &#x2705; Yes |
 | [RedisTimeSeries]({{<relref "/modules/redistimeseries">}}) | &#x2705; Yes | &#x2705; Yes |
 | [RedisBloom]({{<relref "/modules/redisbloom">}}) | &#x2705; Yes | &#x2705; Yes |
 | [RedisGears]({{<relref "/modules/redisgears">}}) | &#x2705; Yes | &#x274c; No |
 | [RedisAI]({{<relref "/modules/redisai">}})       | &#x2705; Yes | &#x274c; No |
 
-[^4]: Redis Enterprise Cloud currently supports Redis Stack for Fixed and Free subscriptions only.  For details, see [Create database]({{< relref  "/rc/databases/create-database#fixed-and-free-module-options" >}}).
+[^4]: At this time, RedisGraph cannot be combined with other modules in multi-sharded databases.
```
Comment (rrelledge) [inline content/modules/enterprise-capabilities.md:29]:
```suggestion
[^4]: RedisGraph cannot be combined with other modules in multi-sharded databases.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a couple of small suggested changes.

File: content/modules/enterprise-capabilities.md (hunk 1 above)
Comment (lanceleonard) [reply content/modules/enterprise-capabilities.md]: Fixed.

File: content/modules/_index.md [hunk 2]
```diff
@@ -10,11 +10,9 @@ aliases: /modules/
 ---
 Redis develops several modules that extend the core Redis feature set. Some of the features these modules provide include [querying, indexing and full-text search]({{< relref "/modules/redisearch" >}}), [JSON support]({{< relref "/modules/redisjson" >}}), and [probabalistic data structures]({{< relref "/modules/redisbloom" >}}).
 
-[Redis Stack]({{< relref "/modules/redis-stack" >}}) lets you install and leverage modules quickly; it enables access to multiple modules in a single Redis database.  
+[Redis Stack](https://redis.io/docs/stack/) lets you install and leverage modules quickly; it enables access to multiple modules in a single Redis database.  
 
-You can use these modules with [Redis Enterprise Software]({{< relref "/rs" >}}).
-
-You can use many, but not all, modules with [Redis Enterprise Cloud]({{< relref "/rc" >}}).  For details, see [Redis Enterprise module support]({{<relref "modules/enterprise-capabilities#redis-enterprise-module-support">}}).
+[Redis Enterprise Software]({{< relref "/rs" >}}) and [Redis Enterprise Cloud]({{< relref "/rc" >}}) support all capabilities of [Redis Stack]({{< relref "/modules/redis-stack" >}})
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/_index.md]: Fixed.

File: content/modules/redis-stack.md [hunk 3]
```diff
@@ -9,27 +9,29 @@ aliases: /modules/redis-stack/
          /modules/redis-stack.md
 ---
 
-[Redis Stack](https://redis.io/docs/stack) enables new data types and use cases for Redis databases.
+[Redis Stack](https://redis.io/docs/stack) enables multiple modules to be used with open source Redis databases; it simplifies installation and deployment.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/redis-stack.md]: Reworked.

File: content/modules/redis-stack.md [hunk 4]
```diff
@@ -9,27 +9,29 @@ aliases: /modules/redis-stack/
...
+- [RedisJSON]({{< relref "/modules/redisjson/" >}})
+- [RedisGraph]({{< relref "/modules/redisgraph/" >}})
+- [RedisTimeSeries]({{< relref "/modules/redistimeseries/" >}})
+- [RedisBloom]({{< relref "/modules/redisbloom/" >}})
 
-Each module is available to the database, which means you can combine their capabilities to meet your needs.
+Redis Enterprise Software and Redis Enterprise Flexible/Annual subscriptions also support all capabilities of Redis Stack.  Here, you select the specific modules that support your specific scenario.
 
-Redis Stack is not supported by Redis Enterprise Software at this time.
+Each module is available to the database and you can combine their capabilities to meet your needs.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/redis-stack.md]: Fixed.

File: content/modules/redis-stack.md [hunk 5]
```diff
@@ -9,27 +9,29 @@ aliases: /modules/redis-stack/
...
 ---
 
-[Redis Stack](https://redis.io/docs/stack) enables new data types and use cases for Redis databases.
+[Redis Stack](https://redis.io/docs/stack) enables multiple modules to be used with open source Redis databases; it simplifies installation and deployment.
 
-Redis Enterprise Cloud supports Redis Stack through the [Create database]({{< relref "/rc/databases/create-database" >}}) workflow for Fixed and Free subscriptions.
+Redis Enterprise Cloud and Redis Enterprise Software support all capabilities of [Redis Stack](https://redis.io/docs/stack).
 
-When you create a new database in a Fixed or Free Redis Enterprise Cloud subscription, you can set the **Type** property to _Redis Stack_.  
+For Fixed for Free subscriptions, Redis Enterprise Cloud supports Redis Stack through the [Create database]({{< relref "/rc/databases/create-database" >}}) workflow.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/redis-stack.md]: Fixed.

File: content/modules/enterprise-capabilities.md (hunk 1 above)
Comment (banker) [reply content/modules/enterprise-capabilities.md]: Is the term "multi-sharded" or just "sharded"?

Comment (lanceleonard) [reply content/modules/enterprise-capabilities.md]: Fixed.

**Pull request merged**